Mathlib/Analysis/NormedSpace/OperatorNorm.lean

```diff
@@ -981,22 +971,9 @@ def prodMapL : (M₁ →L[𝕜] M₂) × (M₃ →L[𝕜] M₄) →L[𝕜] M₁
       apply funext
       rintro ⟨φ, ψ⟩
       refine' ContinuousLinearMap.ext fun ⟨x₁, x₂⟩ => _
-      -- Porting note: mathport suggested:
-      -- ```
-      -- simp only [add_apply, coe_comp', coe_fst', Function.comp_apply, compL_apply, flip_apply,
-      --   coe_snd', inl_apply, inr_apply, Prod.mk_add_mk, add_zero, zero_add, coe_prodMap'
-      --   Prod_map, Prod.mk.inj_iff, eq_self_iff_true, and_self_iff]
-      -- rfl
-      -- ```
-      -- Frustratingly, in `mathlib3` we can use:
-      -- ```
-      -- dsimp   -- ⊢ (⇑φ x.fst, ⇑ψ x.snd) = (⇑φ x.fst + 0, 0 + ⇑ψ x.snd)
-      -- simp
-      -- ```
-      -- Here neither `dsimp` or `simp` seem to make progress.
-      repeat first | rw [add_apply] | rw [comp_apply] | rw [flip_apply] | rw [compL_apply]
-      simp only [coe_prodMap', Prod_map, coe_fst', inl_apply, coe_snd', inr_apply, Prod.mk_add_mk,
-        add_zero, zero_add])
+      -- We have to use `simp [(X)]` to avoid unification of instance implicit arguments
+      -- which only unify at `.default` reducibility
+      simp [(add_apply), (comp_apply), (flip_apply), (compL_apply)])
```

Comment:
What semireducible definitions need to be unfolded?

Reply:
Lean doesn't seem to want to unfold `PseudoMetricSpace.toUniformSpace`.

Reply:
Actually this might be another case of `RingHom.id` not being reducible. See[Simp claims not defeq but rfl works](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/Simp.20claims.20not.20defeq.20but.20rfl.20works)

Reply:
No that doesn't help anything.

Reply:
We end up at 
```lean
 ❌ @Exists =?= PseudoMetricSpace.toUniformSpace.1.1 
```